packages/design-system/src/components/ButtonIcon/ButtonIcon.stories.mdx

```diff
@@ -0,0 +1,183 @@
+import { ArgsTable, Meta, Canvas, Story } from '@storybook/addon-docs';
+import { FigmaImage, FigmaLink, Links } from '../../docs';
+import ButtonIconPrimitive from './Primitive/ButtonIconPrimitive';
+import * as Stories from './ButtonIcon.stories';
+import InlineMessage from '../InlineMessage/index';
+
+<Meta title="Components/ButtonIcon" component={ButtonIconPrimitive} />
+
+<Links>
+	<FigmaLink href="https://www.figma.com/file/KuJ1PlP77uyXlfOhdniqsZ/ButtonIcon?node-id=1%3A75" />
+</Links>
+
+# ButtonIcon
+
+This component should be used when icons are meant to be clicked on.
+
+It handles the two largest usecases we have for clickable icons: actions and toggles.
+
+- Always try use when an icon is meant to be clicked on.
+- Always provide developers with copy for the tooltip.
+- If the button holds the ON / OFF state of something, then use the Toggle variants.
+- If the button is floating on top of something else, use the Floating variant.
+- Only use size XS sporadically.
+- Spacing between buttons should be at least 4px (XXS), 8px is best (XS).
+
+## Zoning
+
+<FigmaImage
+	src="https://www.figma.com/file/qc8oUSyVJM67gSsbTShftt/%F0%9F%93%90-Zoning?node-id=1049%3A415"
+	alt="zoning image for ButtonIcons"
+/>
+
+## Style
+
+### Variations
+
+<FigmaImage
+	src="https://www.figma.com/file/KuJ1PlP77uyXlfOhdniqsZ/ButtonIcon?node-id=3232%3A34023"
+	alt="variations image for ButtonIcon"
+/>
+
+#### Default
+
+This is the go-to ButtonIcon for most generic actions.
+
+```tsx
+import { ButtonIcon } from '@talend/design-system';
+
+function MyComponent() {
+	return (
+		<>
+			<ButtonIcon onClick={onAddDataset()}>Add a dataset</ButtonIcon>
+			<ButtonIcon onClick={onAddDataset()} size="S" disabled>
+				Add a dataset
+			</ButtonIcon>
+			<ButtonIcon onClick={onAddDataset()} size="XS">
+				Add a dataset
+			</ButtonIcon>
+			<ButtonIcon onClick={onAddDataset()} isLoading>
+				Add a dataset
+			</ButtonIcon>
+		</>
+	);
+}
+```
+
+Component not yet built
+
+<Canvas>
+	<Story story={Stories.Default} />
+</Canvas>
+
+#### Toggle
+
+If the button needs to hold an active / inactive state, then it is a Toggle.
+
+Don’t use Toggles in series, opt for badge dropdowns instead.
+
+```tsx
+import { ButtonToggle } from '@talend/design-system';
+
+function MyComponent() {
+	return (
+		<>
+			<ButtonToggle isActive={false} onClick={onToggleEditMode()}>
+				Edit this dataset
+			</ButtonToggle>
+			<ButtonToggle isActive={false} onClick={onToggleEditMode()} size="S" disabled>
+				Edit a dataset
+			</ButtonToggle>
+			<ButtonToggle isActive={false} onClick={onToggleEditMode()} size="S" isLoading>
+				Edit a dataset
+			</ButtonToggle>
+		</>
+	);
+}
+```
+
+Component not yet built
+
+<Canvas>
+	<Story story={Stories.Toggle} />
+</Canvas>
+
+#### Floating
```

Comment:
Without having access to the Figma, i don't see how it has to be used instead of the other ones

Reply:
Alright, I'll try to be more descriptive.

Basically it's for TPD and other "playground" deals, when you have buttons that are sticky on top of other content. It's an elevation buff.

Reply:
Exactly, or the "+" on top of the lines between each node.

Reply:
Thanks for the clarification